dbt/models/iasworld/schema/iasworld.owndat.yml

```diff
@@ -91,6 +95,8 @@ sources:
             description: Code for predefined note
           - name: own1
             description: '{{ doc("shared_column_owner_name") }}'
+
+
```

Comment:
**[Question, non-blocking]** Any particular meaning to these newlines?

Reply:
Fixed in 70b5902!